docs/docs/deploy/deploy-rasa-plus.mdx

```diff
@@ -13,43 +13,9 @@ import RasaProBanner from '@theme/RasaProBanner';
 
 To use the Rasa Plus image in a Helm deployment, you will need to substitute `rasa` with `rasa-plus` as
 the image name. You will also need to pass your license in an env var
-and [specify a pull secret for the image](#creating-a-pull-secret).
+and [specify a pull secret for the image](#step-1---create-a-pull-secret).
 
-For the [Rasa Helm Chart](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa),
-that means your values should be updated as follows:
-
-```yaml
-registry: <registry-for-rasa-plus-image>
-command:
-  - rasa
-image:
-  name: rasa-plus
-  tag: 0.1.1
-  pullSecrets:
-    - name: rasa-plus-pull-secret
-extraEnv:
-  - name: RASA_PRO_LICENSE
-    value: <your-license-string>
-```
-
-For the [Rasa Enterprise Helm Chart](https://github.com/RasaHQ/rasa-x-helm), update
-your values as follows:
-
-```yaml
-images:
-  imagePullSecrets:
-    - name: rasa-plus-pull-secret
-rasa:
-  command:
-    - rasa
-  name: <registry-for-rasa-plus-image>
-  tag: 0.1.1
-  extraEnv:
-    - name: RASA_PRO_LICENSE
-      value: <your-license-string>
-```
-
-#### Creating a Pull Secret
+## **Step 1 - create a pull secret**
```

Comment:
```suggestion
## **Step 1: image pull secret**
```
uses a different delimiter after the step compared to other headlines, should be consistent